composer.json

```
@@ -1,24 +1,30 @@
 {
-    "name": "ifsnop/mysqldump-php",
+    "name": "druidfi/mysqldump-php",
     "description": "PHP version of mysqldump cli that comes with MySQL",
     "type": "library",
-    "keywords": ["mysql", "mysqldump", "pdo", "php7", "php5", "database", "php", "sql", "hhvm", "mariadb", "mysql-backup"],
-    "homepage": "https://github.com/ifsnop/mysqldump-php",
+    "keywords": ["mysql", "mysqldump", "pdo", "php7", "php8", "database", "php", "sql", "mariadb", "mysql-backup"],
+    "homepage": "https://github.com/druidfi/mysqldump-php",
     "license": "GPL-3.0-or-later",
     "minimum-stability": "stable",
     "authors": [
+        {
+            "name": "Druid.fi",
+            "email": "info@druid.fi"
+        },
         {
             "name" : "Diego Torres",
             "homepage": "https://github.com/ifsnop",
             "role": "Developer"
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": "^7.4 || ^8.0",
+        "composer-runtime-api": "^2",
+        "ext-pdo": "*"
     },
     "require-dev": {
-        "squizlabs/php_codesniffer": "1.*",
-        "phpunit/phpunit": "4.8.36"
+        "squizlabs/php_codesniffer": "3.*",
+        "phpunit/phpunit": "^8.5.15 || ^9"
     },
     "autoload": {
         "psr-4": {"Ifsnop\\": "src/Ifsnop/"}
```

```suggestion
        "psr-4": {"Ifsnop\\Mysqldump\\": "src/"}
```

And move `src/Ifsnop/Mysqldump/` to `src/`.

Deep directory tree style was used in PSR-0.

---

Hi, `main` branch (which is default in this repo) has already:

```
"require-dev": {
        "squizlabs/php_codesniffer": "3.*",
        "phpunit/phpunit": "^8.5.15 || ^9"
    },
```

Could you create an issue about the move from src/Ifsnop/Mysqldump/ to src/ ? I opened the issues section.

---

My suggested change is only for `"psr-4"` line, which is necessary for move from src/Ifsnop/Mysqldump/ to src/.

php_codesniffer and phpunit is unrelated here.